core/src/main/java/org/infinispan/persistence/manager/PersistenceManager.java

```diff
@@ -235,6 +268,8 @@ public boolean test(StoreConfiguration configuration) {
 
    void setClearOnStop(boolean clearOnStop);
 
+   void writeToAllNonTxStoresSync(MarshallableEntry marshalledEntry, int segment, Predicate<? super StoreConfiguration> predicate);
```

Comment:
Looks like I forgot to deprecate this one and add the comments like the other methods.